README.md

```diff
@@ -416,6 +416,24 @@ ssh_sg:
     - WebAccessSecurityGroup
 ```
 
+### AWS IIC/SSO Group IDs
+
+Looks up AWS Identity Center group name in the configured Identity Store and returns the ID suitable for use in AWS IIC assignments.
+It is likely that account and role will need to be specified to do the lookup.
+
+In stack_master.yml
+
+```yaml
+sso_identity_store_id: `d-12345678`
```

Comment:
I'm not convinced this needs to be a top-level attribute.

Reply:
There's prior art of embedding multiple attributes in the parameter value:

```yaml
bucket_name:
  # Output from a stack in a different region
  stack_output: us-east-1:init-bucket/bucket_name
```

Could we do the same here?

```yaml
GroupId:
  sso_group_id: 'd-1234567df8:Okta-App-AWS-Group-name'
  role: service-role/rolename
  account: '123456789012'
```

Reply:
It would be _super_ unusual to need to refer to 2 different identity stores so it felt like you'd just be copying the same thing over and over and over again.

Reply:
The idea that the parameter is looked up by attributes outside of what is provided in the parameter file is surprising. Additionally, the configuration values are stored in separate locations, often in different files. This distance in configuration increases the potential for readers to be confused.

While nesting the identity store ID in the parameter value will increase duplication, the lookup will be explicit and more understandable for engineers.

Reply:
I've converted across to this format as it made it easy to specify the region the directory store is in,  in our case it's the same as where we create the assignments, but that is not necessarily the case.

Reply:
[f55d905](https://github.com/envato/stack_master/pull/390/commits/f55d90506d8ce6f4ae7494cb9d4991fb0cf2903c)
[f702341](https://github.com/envato/stack_master/pull/390/commits/f702341b34676c2bbe81ddfa40ef47edd88e8787)
[af6e6bd](https://github.com/envato/stack_master/pull/390/commits/af6e6bdda9546da80a9842e30963a5eb9b453b8f)